notebooks/working/_03-tb-deconfounder-predictive-checks.py

```diff
@@ -594,22 +336,22 @@ def perform_visual_predictive_cit_test(
 # As before, we can immediately expect the posterior predictive version of the conditional independence to fail since the observed data is generally unlike the sampled data.
 # This is dissimilarity is, a-priori, expected to remain in the conditional independence test.
 
+# ### Perform posterior-predictive conditional independence test
+
 # +
-# Get the prior predictive values for the test
+# Get the posterior predictive values for the test
 posterior_samples_triplet =\
     np.concatenate((x_samples_post[:, cols_for_test, :],
                     z_samples_post),
                    axis=1)
 
 # Test out the predictive conditional independence test
 post_pval, post_sampled_pvals, post_obs_pvals =\
-    perform_visual_predictive_cit_test(
+    li.perform_visual_predictive_cit_test(
         posterior_samples_triplet,
         obs_sample)
 # -
 
-sbn.kdeplot(post_sampled_pvals)
-
 print(post_obs_pvals)
 
 # # Test the predictive conditional independence tests
```

Comment:
Could this go at the top of the notebook?

Reply:
I just got rid of this altogether because the kdeplot wasn't necessary (it was just for my original viewing pleasure). Are you referring to something else going at the top of the notebook?